src/transfer-manager.ts

```diff
@@ -105,6 +111,7 @@ export interface DownloadFileInChunksOptions {
   chunkSizeBytes?: number;
   destination?: string;
   validation?: 'crc32c' | false;
+  noReturnData?: true;
```

Comment:
To prevent this from altering existing behavior, i.e. backwards compatible change it will need to default to false.

Reply:
Also we will need to add this to the JSDoc fields above the `downFileInChunks` method.

Reply:
Ah, I was just going with no value at all or `true` so existing users won't be affected? (Sort of matching the style of the `validation` field above.)

Reply:
Being as we currently return data, albeit the incorrect type per the definition, we want this to default to returning the data to maintain that compatibility.